DateTimeInput/README.md

```diff
@@ -0,0 +1,74 @@
+# DateTimeInput Interface
+
+**Status**: Accepted
+
+| Name         | ID                                                                |
+| :--------    | :---------------------------------------------------------------- |
+| DateTimeInput| 4e862a9df81183ab425bdf0fbd76bd0b558c7f44c24887b4354bf1c26c74a623  |
+
+
+## Description
+
+Allows to input date and time.
+
+`getDate` - returns date timestamp selected by user.
+
+Note: function accepts date in unixtime (seconds from the midnight UTC-0 of 1st Jan 1970) which is int128 bits long, negative values are allowed. Date unixtime includes day, month, year and 0 hours, 0 minutes and 0 seconds.
+
+arguments:
+
+	answerId: uint32 - function id of result callback
+
+    prompt: bytes - utf-8 string to print to the user before input.
+
+    minDate: int128 - minimum date that can be chosen.
+    
+    maxDate: int128 - maximum date that can be chosen.
+
+returns:
+
+	date: int128 - chosen date unixtime (UTC-0).
+
+`getTime` - returns the timestamp within a day selected by the user.
+
+arguments:
+
+	answerId: uint32 - function id of result callback
+
+    prompt: bytes - utf-8 string to print to the user before input.
+
+    minTime: uint32 - minimum time that can be chosen.
+    
+    maxTime: uint32 - maximum time that can be chosen.
+
+    interval: uint8 - minimal allowed interval between neibour times in minutes. Valid values are {1, 2, 3, 4, 5, 6, 10, 12, 15, 20, 30}.
```

Comment:
Why are there such valid values?

Reply:
> Why are there such valid values?

Because they are divisors of 60. Hence this is rather convenient in terms of UX when picking the time (e.g. when using any components for the minutes scroll)

Reply:
In that case should we add 60? 
what about 120,180, 240, 360, 480, 720? they are divisors of 1440 (24*60)

Reply:
I’d say no, as no known and commonly used interface suggests such values, only the listed above. I can show a couple of libraries which use exactly the same in order to let pick any hour (cannot skip them) and let at least pick 0 or 30 minutes. 

example: https://developer.apple.com/documentation/uikit/uidatepicker/1615998-minuteinterval